rs/consensus/src/consensus/validator.rs

```diff
@@ -992,6 +1080,35 @@ impl Validator {
                 continue;
```

Comment:
Would it make sense to send out equivocation proofs already before the rank-based delay has passed?

Reply:
done